tests/trainer/resources/fashion_mnist_mpi.py

```
@@ -0,0 +1,116 @@
+
+    from torchvision import datasets, transforms
+
+    try:
+        dataset = datasets.FashionMNIST(
+            dataset_path,
+            train=True,
+            download=False,
+            transform=transforms.Compose([transforms.ToTensor()]),
+        )
+        logger.info("[Rank %s/%s] Using local FashionMNIST dataset at %s", rank, size, dataset_path)
+    except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)